src/bundle/Resources/views/themes/admin/ui/layout.html.twig

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <meta name="CSRF-Token" content="{{ csrf_token('authenticate') }}" />
         <meta name="SiteAccess" content="{{ app.request.get('siteaccess').name }}" />
-        <meta name="UserId" content="{{ ibexa_admin_ui_config.user.user.id }}" />
+        <meta name="UserId" content="{{ ibexa_admin_ui_config.user.user.id|default() }}" />
```

Comment:
this looks little strange, that in twig for admin panel there can be no authenticated user, but I guess it has sth to do with corporate account and it's necessary evil

Reply:
Wouldnt that be anonymous user id anyway?
@Nattfarinn

Reply:
Nope. `user` is null.